plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerDelegate.java

```diff
@@ -41,6 +41,11 @@
 
 @SuppressWarnings("restriction") // For FacetUtil
 public class LocalAppEngineServerDelegate extends ServerDelegate {
+  public static final String RUNTIME_TYPE_ID =
+      "com.google.cloud.tools.eclipse.appengine.standard.runtime";
```

Comment:
Oh geesh, this is quite circular.  `.localserver` imports classes from `.facets`, but `.facets` uses runtime and server definitions from `.localserver`.